frontend/src/app/organization/organization-details/organization-details.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+
+import { Organization } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-details',
+  templateUrl: './organization-details.component.html',
+  styleUrls: ['./organization-details.component.scss'],
+})
+
+export class OrganizationDetailsComponent {
+  @Input() djangoRoot: string;
```

Comment:
Do we need to pass this `djangoRoot` variable down the component chain?

Maybe instead of that do:
```
import { environment } from '../environments/environment';
```
and inside component class (or in template, because You need to resolve organization observable):
```
public editUrl = `${environment.djangoRoot}/organizations/{{ organization.slug }}/{{ organization.id }}/edit`;
public createUrl = `${environment.djangoRoot}/offers/create`;
```

Reply:
I wanted to make organization root component "to know" everything when child components only those whats necessary. I think djangoRoot is going to be used in the component with organization's list of offers too, so maybe it's not so bad idea to have this variable set in organization root component and to pass its value to child components.